.redocly.lint-ignore.yaml

```diff
@@ -0,0 +1,41 @@
+# This file instructs Redocly's linter to ignore the rules contained for specific parts of your API.
+# See https://redoc.ly/docs/cli/ for more information.
+swagger.yml:
```

Comment:
Can this include more comments/docs as to why this is configured as is? Why do we need ignores?